modules/katello_sync_plan.py

```diff
@@ -192,12 +190,8 @@ def sync_plan(self, name, organization, interval=None, enabled=None, sync_date=N
 
 
 def main():
-    module = AnsibleModule(
+    module = ForemanAnsibleModule(
```

Comment:
KatelloAnsibleModule?

Reply:
Ah i see, there is no such thing...
But should this be an entity anyway? If yes, let's leave it to another PR.

Reply:
To dry up that `organization=dict(required=True),` in here?

Reply:
and no, this is not an entity, as it has enabled/disabled vs present/absent (don't ask)

Reply:
Can't it be present and disabled? Now i just have to ask.

Reply:
I have no idea, I didn't write the module. Maybe @akofink wants to chime in, but this is probably out of scope for this PR :)

Reply:
The [example](https://github.com/theforeman/foreman-ansible-modules/blob/master/modules/katello_sync_plan.py#L78) shows present and disabled. It doesn't seem like it's possible to *remove* a sync plan using this module, however, so I'd agree there's room for improvement here!